tests/Microsoft.Identity.Test.Unit/ManagedIdentityTests/ServiceFabricTests.cs

```diff
@@ -0,0 +1,195 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Globalization;
+using System.Net;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client;
+using Microsoft.Identity.Test.Common;
+using Microsoft.Identity.Test.Common.Core.Helpers;
+using Microsoft.Identity.Test.Common.Core.Mocks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Identity.Test.Unit.ManagedIdentityTests
+{
+    [TestClass]
+    public class ServiceFabricTests
+    {
+        private const string Endpoint = "http://localhost:40342/metadata/identity/oauth2/token";
```

Comment:
Isn't this Azure ARC endpoint?

Reply:
Yes, it is just unit test so I did not change it. Main thing is that the endpoint is fetched from the environment variable.